docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -186,9 +187,21 @@ public DockerClientExecutor(CubeDockerConfiguration cubeConfiguration) {
         configBuilder.withDockerTlsVerify(cubeConfiguration.getTlsVerify());
 
         this.dockerClientConfig = configBuilder.build();
-        this.cubeConfiguration = cubeConfiguration;
 
         this.dockerClient = buildDockerClient();
+
+
```

Comment:
Rather than exposing this outside it made sense to me to add a shutdown hook for this important cleanup.